server/src/analyzer/binder.ts

```diff
@@ -25,23 +25,79 @@ import { convertOffsetsToRange } from '../common/positionUtils';
 import { PythonVersion } from '../common/pythonVersion';
 import { getEmptyRange } from '../common/textRange';
 import { TextRange } from '../common/textRange';
-import { ArgumentCategory, AssertNode, AssignmentExpressionNode, AssignmentNode,
-    AugmentedAssignmentNode, AwaitNode, BinaryOperationNode, BreakNode,
-    CallNode, ClassNode, ContinueNode, DelNode, ExceptNode, ExpressionNode, ForNode,
-    FunctionNode, GlobalNode, IfNode, ImportAsNode, ImportFromNode, LambdaNode,
-    ListComprehensionNode, MemberAccessNode, ModuleNameNode, ModuleNode, NameNode, NonlocalNode,
-    ParseNode, ParseNodeType, RaiseNode, ReturnNode, StatementNode, StringListNode,
-    SuiteNode, TernaryNode, TryNode, TypeAnnotationNode,
-    UnaryOperationNode, WhileNode, WithNode, YieldFromNode, YieldNode } from '../parser/parseNodes';
+import {
```

Comment:
Is there a way to configure prettier to not format each import on its own line? This looks really bad to me.